implants/lib/eldritch/src/sys/get_os_impl.rs

```diff
@@ -1,58 +1,80 @@
 use anyhow::Result;
 use starlark::collections::SmallMap;
 use starlark::const_frozen_string;
-use starlark::values::Heap;
 use starlark::values::dict::Dict;
+use starlark::values::Heap;
 
 #[derive(Debug)]
 struct OsInfo {
-    arch:           String,
-    desktop_env:    String,
-    distro:         String,
-    platform:       String,
+    arch: String,
+    desktop_env: String,
+    distro: String,
+    platform: String,
 }
 
 pub fn get_os(starlark_heap: &Heap) -> Result<Dict> {
-
     let cmd_res = handle_get_os()?;
 
     let res = SmallMap::new();
     let mut dict_res = Dict::new(res);
     let arch_value = starlark_heap.alloc_str(&cmd_res.arch);
-    dict_res.insert_hashed(const_frozen_string!("arch").to_value().get_hashed().unwrap(), arch_value.to_value());
```

Comment:
If we're gonna split all of these onto new lines I'd prefer we switch this to a macro since it's mostly boilerplate / doesn't change between instances of us inserting into a dict.